ui-components/src/locales/general.json

```diff
@@ -1497,7 +1497,7 @@
       "62plus": "62+"
     },
     "disability": {
-      "prompt": "Do you have a disability?",
+      "prompt": "Does anyone in your household have a disability?",
```

Comment:
Looks like this breaks the unit tests. :disappointed:

Reply:
Yep just pushed a fix to update the tests!